### PR TITLE
Support alternative (smarter) OFM vol-format

### LINKED ISSUE #285: [ofmvol2csv] Support different data  layout from OFM
Currently, ofmvol2csv supports the format

```
*METRIC
*DAILY
*DATE           *OIL    *GAS           *WATER  *GINJ    *WINJ   *THP    *BHP  
*NAME 99/6-15
01.01.1991  0.00    0.0000          0.00    0.00    0.00    0.00    0.00  

```
(repeated for each well)

while it should also support (the easier to parse) format: 
```
*METRIC
*DAILY
*WELL           *DATE           *OIL    *GAS           *WATER  *GINJ    *WINJ   *THP    *BHP  
99/6-15         01.01.1991  0.00    0.0000          0.00    0.00    0.00    0.00    0.00  
99/9-X-27H  01.01.1991  0.00    0.0000          0.00    0.00    0.00    0.00    0.00  
99/9-X-51H  01.01.1991  0.00    0.0000          0.00    0.00    0.00    0.00    0.00  
99/9-26         01.01.1991  0.00    0.0000          0.00    0.00    0.00    0.00    0.00  
99/9-X-27AH 09.10.1991  571.00  86250.00    1.00    0.00    0.00    0.00    140.00
99/9-X-27AH 10.10.1991  1672.00 253750.00   2.00    0.00    0.00    0.00    143.00
```

### PR DESCRIPTION
**Pull request by berland**
Resolves #285

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: src/subscript/ofmvol2csv/ofmvol2csv.py [hunk 1]
```diff
@@ -278,18 +307,25 @@ def process_volstr(volstr):
...
+            if not wellframe.empty:
+                wellframes.append(wellframe)
+        if wellframes:
+            return pd.concat(wellframes, sort=False).sort_index()
+        return pd.DataFrame()
+
+    # For the OFM syntax with WELL as a table attribute:
+    data_start_row = [idx for idx, line in enumerate(filelines) if "WELL" in line][0]
+    data = parse_ofmtable(filelines[data_start_row:], columnnames)
+    return data
```
Comment (wouterjdb) [inline src/subscript/ofmvol2csv/ofmvol2csv.py:328]: Maybe change to only 1 return statement for readability?

Comment (berland) [reply src/subscript/ofmvol2csv/ofmvol2csv.py]: Done.

**Review by wouterjdb — APPROVED** [review]
LGTM 👍

**Pull request merged**